GNUmakefile

```
@@ -621,6 +621,8 @@ smoketest: testchdir
 	@$(MD5SUM) test.hex
 	$(TINYGO) build -size short -o test.hex -target=feather-rp2040      examples/device-id
 	@$(MD5SUM) test.hex
+	$(TINYGO) build -size short -o test.hex -target=pico2-ice	    examples/blinky1
```

This is really a nitpick (sorry!) but can you make sure the alignment here is like the other smoke tests?

```suggestion
	$(TINYGO) build -size short -o test.hex -target=pico2-ice	        examples/blinky1
```